tests/lib/cli.js

```diff
@@ -1112,4 +1112,11 @@ describe("cli", () => {
         });
     });
 
+    describe("when passing --info", () => {
+        it("should print out debugging information concerning the local environment", async() => {
+            await cli.execute("--info");
```

Comment:
Does `cli.execute` return a promise? I thought it was synchronous. This is why I don't think we should have a dangling promise here - there is no way for the caller to handle the promise correctly.